docker-compose.yml

```diff
@@ -4,6 +4,19 @@
 # Please consult its README.md for bucket creation steps
 version: "3"
 services:
+  validator:
+    image: quay.io/cloudservices/playbook-artifact-validator
+    build:
+      context: .
+    links:
+    - kafka
+    - minio
+    ports:
+      - 9001:9001
```

Comment:
[The spec](https://github.com/compose-spec/compose-spec/blob/master/spec.md#ports) recommends writing this as `'9001:9001'`, to avoid yaml interpretation issues:

> `HOST:CONTAINER` SHOULD always be specified as a (quoted) string, to avoid conflicts with [yaml base-60 float](https://yaml.org/type/float.html).